xbmc/Application.cpp

```diff
@@ -2687,6 +2687,12 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
       m_skipGuiRender = true;
 #endif
 
+    if (g_advancedSettings.m_guiSmartRedraw && m_guiRefreshTimer.IsTimePast())
```

Comment:
Brackets about bit comparision missing

Reply:
no brackets missing here, this is just a boolean and operation